recipe/meta.yaml

```diff
@@ -52,7 +57,7 @@ about:
     command line tools. It is primarily meant to be run inside existing applications. That said, 
     APScheduler does provide some building blocks for you to build a scheduler service or to run a 
     dedicated scheduler process.
-  doc_url: https://apscheduler.readthedocs.io
+  doc_url: https://apscheduler.readthedocs.io/en/3.x/
```

Comment:
`https://apscheduler.readthedocs.io` is fine. The redirect is not a critical issue for the `readthedocs` site

Reply:
Okay, thanks! So, should I change it back? Or leave it as is, but make note of it for future reference?

Reply:
It's better to change it back